presto-docs/src/main/sphinx/release/release-0.282.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.282
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix ``TEMPORARY`` definition :doc:`/sql/create-function` and :doc:`/sql/drop-function`.
+* Fix a bug where ``cardinality(map_keys(x))`` and ``cardinality(map_values(x))`` would return wrong results.
+* Fix Presto builds on ARM based processors.
```

Comment:
We should not add this note in this release since it's not completely done yet, reference: #19981

Reply:
Thank you!